### ISSUE TITLE
Pre-training, when to start?

### ISSUE THREAD

**Issue by liqianxi** [liqianxi/DST_MTRL#2]
We might not want to enable masked-network immediately, may be 20 episodes pretraining?
